_posts/2018-07-11-cockpit-172.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Cockpit 172
+author: pitti
+date: 2018-07-11
+tags: cockpit linux
+slug: cockpit-172
+category: release
+summary: Cockpit with on-demand PCP install, flow control, and Python 3 support
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 172.
+
+### System: Offer installation of PCP
+
+The System page now shows an "Enable persistent metrics…" link if the
+[PCP](https://pcp.io/) package is not installed. This is similar to the
+on-demand installation of the NFS client packages in
+[Cockpit 169](https://cockpit-project.org/blog/cockpit-169.html).
+
+![Install PCP on demand](/images/pcp-install-on-demand.png)
+
+### Software Updates: Improve layout in mobile mode
+
+The Software Updates page fixed some incorrect spacing and layout in small
+browser windows and mobile browsers:
```

Comment:
The Software Updates page improves spacing and layout in small browser windows and mobile browsers: